pyvrp/read.py

```diff
@@ -67,8 +70,8 @@ def read(
 
             * ``'round'`` rounds the values to the nearest integer;
             * ``'trunc'`` truncates the values to be integral;
-            * ``'trunc1'`` or ``'dimacs'`` scale and truncate to the nearest
-              decimal;
+            * ``'dimacs'`` scales by 10 and rounds down to an integer;
+            * ``'exact'`` scales by 1000 and rounds to the nearest integer.
```

Comment:
In view of #491, there may be instances that require double precision for which the `exact` round function is slightly off. Solution costs are rounded by 2 decimals, so routes with more than 10 clients can result in slightly different costs.